spec/index.html

```diff
@@ -443,32 +520,68 @@ <h3>Triples</h3>
 
     <ul>
       <li>the <dfn>subject</dfn>, which is an
-      <a>IRI</a> or a <a>blank node</a></li>
+      <a>IRI</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
 
       <li>the <dfn>predicate</dfn>, which is an <a>IRI</a></li>
 
       <li>the <dfn>object</dfn>, which is an <a>IRI</a>,
-      a <a>literal</a> or a <a>blank node</a></li>
+      a <a>literal</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
     </ul>
 
     <p>An RDF triple is conventionally written in the order subject,
       predicate, object.</p>
```

Comment:
Add 
```
<p>RDF triple is usually written <i>triple</i>.<p>
```

Reply:
```suggestion 
    <p>An RDF triple is conventionally written in the order subject,
      predicate, object.</p>
      
    <p>RDF triple is usually written "<a>triple</a>".<p>
```

Reply:
```suggestion
      predicate, object.</p>

    <p>RDF triple is usually written "<a>triple</a>".<p>
```

Reply:
Same could be said of "graph" and "dataset". Note that the exported terms remain `dfn-rdf-triple`, `dfn-rdf-graph`, and `dfn-rdf-dataset`, but we can use various aliasing mechanism internally, such as `data-local-lt`.